prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -1037,3 +1037,39 @@ periodics:
         requests:
           memory: 200Mi
           cpu: 80m
+
+- name: kyma-gke-compass-integration-periodic
+  cron: "00 00 * * *" # "At 00:00 every day"
```

Comment:
Isn't it enough to just put here `0 0 * * *`? IMO it will be more readable.
Also, do we need to run this job on weekends? Maybe `0 0 * * 1-5` would be enough here?

Reply:
done